app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -20,81 +24,109 @@ import DeviceContext from '../context/device';
- * @prop {number} glareScoreThreshold
- * @prop {boolean} isAssessedAsGlare
- * @prop {number} sharpness
- * @prop {number} sharpnessScoreThreshold
- * @prop {boolean} isAssessedAsBlurry
- * @prop {AcuantImageAssessment} assessment
- */
+interface _AcuantImageAnalyticsPayload {
+  documentType: AcuantDocumentTypeLabel;
+  dpi: number;
+  moire: number;
+  glare: number;
+  glareScoreThreshold: number;
+  isAssessedAsGlare: boolean;
+  sharpness: number;
+  sharpnessScoreThreshold: number;
+  isAssessedAsBlurry: boolean;
+  assessment: AcuantImageAssessment;
+}
 
-/**
- * @typedef {ImageAnalyticsPayload & _AcuantImageAnalyticsPayload} AcuantImageAnalyticsPayload
- */
+type AcuantImageAnalyticsPayload = ImageAnalyticsPayload & _AcuantImageAnalyticsPayload;
 
-/**
- * @typedef AcuantCaptureProps
- *
- * @prop {string} label Label associated with file input.
- * @prop {string=} bannerText Optional banner text to show in file input.
- * @prop {string|Blob|null|undefined} value Current value.
- * @prop {(
- *   nextValue: string|Blob|null,
- *   metadata?: ImageAnalyticsPayload
- * )=>void} onChange Callback receiving next value on change.
- * @prop {()=>void=} onCameraAccessDeclined Camera permission declined callback.
- * @prop {'user'|'environment'=} capture Facing mode of capture. If capture is not specified and a
- * camera is supported, defaults to the Acuant environment camera capture.
- * @prop {string=} className Optional additional class names.
- * @prop {boolean=} allowUpload Whether to allow file upload. Defaults to `true`.
- * @prop {ReactNode=} errorMessage Error to show.
- * @prop {string} name Prefix to prepend to user action analytics labels.
- */
+interface AcuantCaptureProps {
+  /**
+   * Label associated with file input
+   */
+  label: string;
+  /**
+   * Optional banner text to show in file input
+   */
+  bannerText: string;
+  /**
+   * Current value
+   */
+  value: string | Blob | null | undefined;
+  /**
+   * Callback receiving next value on change
+   */
+  onChange: (nextValue: string | Blob | null, metadata?: ImageAnalyticsPayload) => void;
+  /**
+   * Camera permission declined callback
+   */
+  onCameraAccessDeclined?: () => void;
+  /**
+   * Facing mode of caopture. If capture is not
+   * specified and a camera is supported, defaults
+   * to the Acuant environment camera capture.
+   */
+  capture: 'user' | 'environment';
+  /**
+   * Optional additional class names
+   */
+  className?: string;
+  /**
+   * Whether to allow file upload. Defaults
+   * to true.
+   */
+  allowUpload?: boolean;
+  /**
+   * Error message to show
+   */
+  errorMessage: ReactNode;
+  /**
+   * Prefix to prepend to user action analytics labels.
+   */
+  name: string;
+}
 
 /**
  * Non-breaking space (`&nbsp;`) represented as unicode escape sequence, which React will more
  * happily tolerate than an HTML entity.
- *
- * @type {string}
  */
-const NBSP_UNICODE = '\u00A0';
+const NBSP_UNICODE: string = '\u00A0';
```

Comment:
this one can be inferred, right? TS will kind of assume from the string literal?
```suggestion
const NBSP_UNICODE = '\u00A0';
```